gateway/fedimint-gateway-server/src/lib.rs

```diff
@@ -180,6 +181,30 @@ impl Display for GatewayState {
     }
 }
 
+/// Helper struct for storing the registration parameters for LNv1 for
+/// the HTTP and Iroh endpoints.
+#[derive(Debug, Clone)]
+struct GatewayRegistrationInfo {
+    pub api: SafeUrl,
```

Comment:
By looking at it, it's unclear what it is exactly, and there's no comment. Name could probably be more descriptive.

I can see below that it will be `iroh://<pubkey>`. Make me wonder if there's a point to even store it, if one could just store pubkey and get the url on flight.